src/reliability.jl

```diff
@@ -0,0 +1,72 @@
+using LinearAlgebra: Diagonal
+
+struct Reliability
+    alpha::Float64
+    dropped::Vector{Pair{Int, Float64}}
+end
+
+function Base.show(io::IO, x::Reliability)
+    println(io, "Reliability for all items: $(round(x.alpha; digits=4))")
+    println(io, "")
+    println(io, "Reliability if an item is dropped:")
+    for i ∈ x.dropped
+        println(io, "item $(i.first): $(round(i.second; digits=4))")
+    end
+end
+
+function _crombach_alpha(covmatrix::AbstractMatrix{T}) where T <: Real
+    k = size(covmatrix, 2)
+    σ = sum(covmatrix)
+    σ_ij = sum(covmatrix - Diagonal(covmatrix)) / (k * (k - 1))
+    ρ = k^2 * σ_ij / σ
+    return ρ
+end
+
+"""
+    crombach_alpha(covmatrix::AbstractMatrix{T}) where T <: Real
+
+Calculate Crombach's alpha (1951) from a covariance matrix `covmatrix` according to
+the Wikipedia formula (https://en.wikipedia.org/wiki/Cronbach%27s_alpha):
+
+```math
+\\rho = \\frac{k^2 \\bar{sigma}_{ij}}{\\sigma^2_X}
+```
+
+where k is the number of items, i.e. columns; σᵢⱼ denote the average of
+the inter-item covariances; and  σ²ₓ consists of item variances and inter-item
+covariances.
+
+Returns a `Reliability` object that holds:
+
+* `alpha`: the reliability score for all items, i.e. columns, in `comatrix`; and
+* `dropped`: A `Vector{Pair{item, score}}` reliability scores if a specific item, i.e. column, is dropped from `comatrix`.
```

Comment:
I would say "variable", as you drop both column and row - right?

Reply:
Here is a great discussion... Generally when you perform a Crombach's alpha (or any other reliability measures) you are using the nomenclature item and latent variable. So I avoided the word "variable" and opted for item and column of the covariance matrix. I think that if we include "variable" in the docstring would bring unavoid confusion.

Reply:
I will add "row" also